### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by Cw3py** [Palakis/obs-ndi#61]
I installed the plugin exactly with the same steps and this plugin doesn't work! I open up my obs and i can't find the plugin, it's not showing up! What can i do?

**Comment by Palakis** [comment]
First of all, be careful with words. This plugin works, but you're encountering issues to set it up.
Furthermore, please provide an OBS log file to help diagnosing and resolving this issue.

**Comment by Palakis** [comment]
You also didn't respect the contribution guidelines, which provide a template of what kind of information to include in a bug report/help request.

**Comment by jakkett** [comment]
Hi, thank you for all your help Palakis. I was also having a similar issue. I installed the NDI plugin on my 1st computer with no issues. But on my 2nd computer it did not install. I tried uninstalling and reinstallng but it wouldn't pop up on the OBS tools section. When I downloaded the NDI files the folder was in the Program Files in my C drive under Newtek. I then noticed that there was also a folder in my documents section named {pf}. That {pf} folder had the NDI plugin files that I needed to move to the root directory of my OBS folder. After moving the files into the OBS folder the NDI plugin showed up in my OBS tools section of my 2nd computer. This may help Cw3py and others who are having the same issues.
